.github/workflows/publish-collection.yml

```diff
@@ -15,8 +15,8 @@ jobs:
         if: github.ref_type != 'tag'
       - name: Github Checkout 🛎
         uses: actions/checkout@v4
-      - name: Check git tag matches version in galaxy.yml
-        run: echo "${{ github.ref_name }}" | grep $(yq -r .version < galaxy.yml)
+      - name: Update galaxy.yml version to git tag
+        run: yq -i '.version = "${{ github.event.release.tag_name }}"' 'galaxy.yml'
```

Comment:
Seems like a bit of a fudge? The change won't be committed.

Reply:
Right, committing that is going to be a painful operation.